### PR TITLE
Resolve:- Enhancement on Application

### PR DESCRIPTION
**Pull request by maharishi1**
Added the left out items:
2. Search functionality inside FAQ
3. Add Chinese translation for Simplified Chinese (Singapore)

Resolves: #884 

Screenshots:- https://drive.google.com/file/d/1WO0XZb4hEGX7_D52GnBPaKlkp7ZAVoCT/view?usp=drivesdk
            https://drive.google.com/file/d/1WHUY_UMPgaF3hlKByqsZ7bCSrcFxQO3c/view?usp=drivesdk

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Run the app and go to FAQ section.

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/res/layout/fragment_faq.xml [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    tools:context=".fragment.FAQFragment">
+    tools:context=".fragment.FAQFragment"
+    android:orientation="vertical">
+
+<!--    <FrameLayout-->
```
Comment (Swati4star) [inline app/src/main/res/layout/fragment_faq.xml:10]: Remove the commented part.
Add a screenshot of layout with the changes

File: app/src/main/res/layout/fragment_faq.xml [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+<!--        android:layout_width="match_parent"-->
+<!--        android:layout_height="wrap_content">-->
+     <EditText
+         android:id="@+id/inputSearch"
+         android:layout_margin="10dp"
+         android:layout_width="match_parent"
+         android:layout_height="wrap_content"
+         android:hint="search....."/>
```
Comment (Swati4star) [inline app/src/main/res/layout/fragment_faq.xml:18]: Don't add strings as constant here. Add them to strings.xml file

**Comment by maharishi1** [comment]
Screenshot from 2020-07-17 08-38-16

**Comment by Swati4star** [comment]
It is not preferable to have 2 different type of changes in single PR. Can you share different PRs for your changes ? @maharishi1

File: app/src/main/java/swati4star/createpdf/adapter/FAQAdapter.java [hunk 3]
```diff
@@ -30,7 +31,7 @@ public FAQAdapter(List<FAQItem> faqs, OnItemClickListener mOnItemClickListener)
      * Inflates the layout view and returns it
      *
      * @param viewGroup - holds view
-     * @param position - view position
+     * @param position  - view position
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/adapter/FAQAdapter.java:34]: remove the formatting changes from this file

File: app/src/main/java/swati4star/createpdf/fragment/FAQFragment.java [hunk 4]
```diff
@@ -26,9 +29,14 @@
...
     private Context mContext;
 
+    @BindView(R.id.inputSearch)
+    EditText inputSearch;
+
     @BindView(R.id.recycler_view_faq)
     RecyclerView mFAQRecyclerView;
 
+
+
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/FAQFragment.java:39]: why these extra lines?

File: app/src/main/java/swati4star/createpdf/fragment/FAQFragment.java [hunk 5]
```diff
@@ -86,4 +126,5 @@ public void onItemClick(int position) {
         faqItem.setExpanded(!faqItem.isExpanded());
         mFaqAdapter.notifyItemChanged(position);
     }
+
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/FAQFragment.java:129]: not needed

File: app/src/main/res/layout/fragment_faq.xml [hunk 6]
```diff
@@ -1,16 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Swati4star) [inline app/src/main/res/layout/fragment_faq.xml:2]: You can revert this file

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -303,7 +302,7 @@
     <string name="font_size_changed">Font Size Changed</string>
     <string name="font_size">Font Size : %1$s</string>
 
-<!--  About US & App Description & Contact US  -->
+    <!--  About US & App Description & Contact US  -->
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml:305]: revert not required changes

**Review by Swati4star — CHANGES_REQUESTED** [review]

**Comment by Swati4star** [comment]
Closing due to inactivity

**Pull request closed without merging**